bin/heal

```diff
@@ -104,6 +104,10 @@ def main():
         VH.send_encodes()
         return None
 
+    # TODO: Data backup
+    # TODO: API key purge
+    # TODO: HTML healthcheck/alert?
```

Comment:
@muzaffaryousaf It's not urgent -- the RDS instance auto-backs up, and the ELB sends alerts via AWS. I'll get rid of that bullet point.